init.rb

```diff
@@ -48,6 +60,14 @@
 
 current_flavor = Rock.flavors.current_flavor
 
+Autoproj.configuration_option 'cxx11', 'boolean',
+:default => 'yes',
+:doc => ["Compile with Cxx11 ? [yes/no]"]
+
+if (Autoproj.user_config('cxx11')) then
+    add_compiler_flag('CXXFLAGS', "-std=c++11")
```

Comment:
This should be in the rock-core package set IMO.

Given that the Rock macros already have a C++11 (CMake) option, it should also be set for them. The support for it is already in rock.core.